src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,107 @@
+import { NavLink } from 'react-router-dom';
+import './Sidebar.scss';
+
+//*import React, { SetStateAction, useContext } from 'react';*//
+//*import classNames from 'classnames';*//
+//*import { Logo } from '';*//
+//*import { FavouritesContext } from '';*//
+//*import { CartContext } from '';*//
+
+
+
+
+type Props = {
+  isMenuOpen: boolean;
+  setIsMenuOpen: React.Dispatch<SetStateAction<boolean>>;
+};
+
+export const SideNavbar: React.FC<Props> = ({ isMenuOpen, setIsMenuOpen }) => {
+  //**const { favourites } = useContext(FavouritesContext);**//
```

Comment:
when the time has come, remember to use favorites not favourites 🤣 . Cheers, mate!

p.s. I needed to write something about your code.